docs/velocity/dev/api/plugin-messaging.mdx

```diff
@@ -17,6 +17,20 @@ flowchart LR
     backend -->|"3 (Incoming)"| Velocity -->|"4 (Outgoing)"| player
 ```
 
+:::warning
```

Comment:
this admonition seems out of place

Reply:
would it be better to be below the mention of the bungeecord plugin channel compat?

as this is probably the most important information on the page, security wise, i would prefer it to remain above the fold, as well as remain emphasized.

Reply:
after some deliberation, I initially thought it'd be better in Case 1, as the first section is more abstract and not really about handling the messages, but then it'd need to be reiterated in Case 3, so it's probably fine as it is

you can make the admonition `danger` for additional emphasis